### ISSUE TITLE
System tray is not accessible when running Rancher Desktop as AppImage

### ISSUE THREAD

**Issue by adamkpickering** [rancher-sandbox/rancher-desktop#2129]
### Actual Behavior

Rancher Desktop does not appear in the system tray when running it as an AppImage. This reduces the functionality of Rancher Desktop, most notably by making it impossible to access Rancher Dashboard.

### Steps to Reproduce

1. Run Rancher Desktop as an AppImage.
2. Note that it isn't in the system tray.

### Result

RD is not in the system tray.

### Expected Behavior

RD should be in the system tray when running as an AppImage.

### Rancher Desktop Version

b65193d0

### Rancher Desktop K8s Version

1.22.7

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

Other Linux

### Operating System / Build Version

Fedora 35 and Rocky Linux 8.5

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

AppImage

**Comment by mook-as** [comment]
See also #824 (this might be a dupe, depending on your point of view).

**Comment by markusgod** [comment]
Can you elaborate what your desktop environment/tray extensions are? I've tested appimage on fedora 36 with https://github.com/ubuntu/gnome-shell-extension-appindicator tray icons, it worked just fine.
![image](https://user-images.githubusercontent.com/20001291/195204001-f2349a7a-6df6-49ef-bc0c-6e95be60346a.png)

**Comment by adamkpickering** [comment]
After some investigation, I discovered that this is a GNOME thing, not an AppImage issue. It seems that the GNOME developers have done away with the concept of a system tray. However, it is possible to get a system tray by installing an extension, for example [Tray Icons](https://extensions.gnome.org/extension/2890/tray-icons-reloaded/) or [AppIndicator](https://extensions.gnome.org/extension/615/appindicator-support/).

Closing this issue since this is normal GNOME behavior.
